src/classes/MAPR_PrimaryAffiliations_TEST.cls

```diff
@@ -574,11 +599,135 @@ private class MAPR_PrimaryAffiliations_TEST {
         System.assertEquals(null, primAffMapResult, 'Invalid Contact field and Record Type mappings should return null when validation is not enabled.');
     }
 
+    /**************************************************************************************************************************************
+    * @description Test to ensure that getModelFromMapping returns a valid model with an error message indicating an invalid lookup field 
+    * mapping when validation is off, an invalid record type is specified and a valid field that does not lookup to Account is mapped.
+    **************************************************************************************************************************************/ 
+    private static void getModelFromMappingInvalidRTAndNonLookupFieldValidationOff(){        
+        MAPR_PrimaryAffiliations_TEST.insertAfflMappingsWithInaccurateRecordTypeNameAndNonLookupPrimaryFieldLabels();
+        MAPR_PrimaryAffiliations maprPrimaryAfflInstance = new STUB_PrimaryAffiliationValidationOff();
+
+        List<Affl_Mappings__c> primaryAfflMappings = UTIL_CustomSettingsFacade.getPrimaryAfflMappings();
+
+        Test.startTest();
+            MODL_PrimaryAfflMapper primaryAfflModel = maprPrimaryAfflInstance.getModelFromMapping(primaryAfflMappings[0]);
+        Test.stopTest();
+
+        String prefix = UTIL_Namespace.StrTokenNSPrefix('');
```

Comment:
This is the first step in applying the prefix and is a superior use of the utilities at our disposal. I would prefer that we use this around the specific String in question getting prefixed.